dapps/v2Explorer/src/constants/Colors.ts

```diff
@@ -0,0 +1,28 @@
+// Will be better used when Themeing is added
```

Comment:
Still keep comment?

Reply:
i can remove it, but my idea is to handle the theme using a context at top of the modal